### PR TITLE
Changes quickstart to look at Maven central as Bintray is no longer used

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Right now, people can't download zipkin anymore. This fixes it by using
central until there is a better way.

Note: This drops GPG verification, as the assumption about who signed
the dist is no longer valid. This is better than scaring people.

Note: There's no requirement to sign the md5 file, even if Bintray did
that by default. ASF projects do not sign the md5.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
I verified this, but would appreciate someone else doing so ASAP as we announced zipkin, but you can't download it until this is merged cc @apache/zipkin-committers 

```bash
$ curl -sSL https://raw.githubusercontent.com/apache/incubator-zipkin-website/9c08e96badab6b35c08143295e515c30a9116cdf/quickstart.sh | bash -s
$ curl -sSL https://raw.githubusercontent.com/apache/incubator-zipkin-website/9c08e96badab6b35c08143295e515c30a9116cdf/quickstart.sh | bash -s io.zipkin.aws:zipkin-autoconfigure-collector-sqs:LATEST:module sqs.jar
```

**Comment by codefromthecrypt** [comment]
fixed the above example with a link to actual file that changed :)

The thing to note is that the server (apache) is 2.13 and the module (still zipkin.io) still works

File: quickstart.sh
Comment (abesto) [inline quickstart.sh:241]: Or even verify only that it's one of the keys in the relevant KEYS file that was used to sign. We don't even necessarily care which one.

**Review by abesto — APPROVED** [review]
Ran both commands, they correctly downloaded the expected jar.

File: quickstart.sh
Comment (codefromthecrypt) [reply quickstart.sh:241]: yep this will work, needs a little logic as still we are mixed mode (ex is it bintray or apache as zipkin-aws for example, is still bintray signed!)

**Comment by codefromthecrypt** [comment]
Thanks @abesto!

**Pull request merged**
